cli/cli.hs

```diff
@@ -458,7 +490,8 @@ vmFromCommand cmd = do
                              , prevRandao
                              )
 
-  contract <- case (cmd.rpc, cmd.address, cmd.code) of
+  codeWrapped <- getCode "--code and --code-file" cmd.codeFile cmd.code
```

Comment:
Here you provide "--code and --code-file" as the message parameter.

Reply:
Yeah, sorry. I removed this message thing, it's not needed.